instances/treasury-devdao.near/widget/pages/settings/index.jsx

```diff
@@ -0,0 +1,27 @@
+return (
+  <div>
+    <Widget
+      src={`${REPL_DEPLOYMENT_ACCOUNT}/widget/components.SidebarAndMainLayout`}
+      props={{
+        leftNavbarOptions: [
+          {
+            title: "Members",
+            href: `${REPL_DEPLOYMENT_ACCOUNT}/widget/pages.settings.MembersPage`,
+            props: {},
+          },
+          // TO BE ADDED LATER
```

Comment:
when

Reply:
Not for a month at least, we will add these pages when we will start the self-creation service.